syntax="proto3"; //消息类型命名规范：使⽤驼峰命名法，⾸字⺟⼤写。
package myprotobuf;

enum MyEnum {
    case0 = 0;
    case1 = 1;
    case2 = 2;
}


message MyCase{
    optional MyEnum  optenum = 1;  // optional
             MyEnum  impenum = 2;  // implicit
}
